java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/SinkWriterStreamObserver.java

```diff
@@ -207,6 +210,17 @@ private void bindSink(
         String connectorName = getConnectorName(sinkParam);
         SinkFactory sinkFactory = SinkUtils.getSinkFactory(connectorName);
         sink = sinkFactory.createWriter(tableSchema, sinkParam.getPropertiesMap());
+        if (connectorName.equals("elasticsearch")) {
```

Comment:
It's unnecessary to set the special schema, since the schema is fixed and we should not access it in es 
 sink.

Reply:
The schema here is for use by the StreamChunkDeserializer, which needs to use our mock schema, not the original one Serialize StreamChunk

Reply:
I see. 

Previously we assume the stream chunk schema is the same as the sink logical schema. Now we have broken the assumption. If so, we think we should have a separated field named like `payload_schema` in the `StartSink` proto. The StreamChunkDeserializer will now use the schema in the field instead of the field in the `SinkParam`.

On rust side, when we send the initial start sink request, for es sink we fill in the special schema, and for other sink we fill in the original sink schema, so that on java side we don't need this special logic.